mautrix_signal/portal.py

```diff
@@ -1930,20 +1934,22 @@ async def update_info(self, source: u.User, info: ChatInfo) -> None:
             await self.update_bridge_info()
             await self.update()
 
-    async def update_expires_in_seconds(self, sender: p.Puppet, expires_in_seconds: int) -> None:
+    async def update_expires_in_seconds(self, sender: p.Puppet, expires_in_seconds: int) -> bool:
         if expires_in_seconds == 0:
             expires_in_seconds = None
         if self.expiration_time == expires_in_seconds:
-            return
+            return False
 
         assert self.mxid
+        self.log.debug(f"Setting portal expiration time to {expires_in_seconds}")
         self.expiration_time = expires_in_seconds
         await self.update()
```

Comment:
This could be not called when handle_signal_group_change is going to save anyway, but can be fixed later too since it's just a minor optimization